ui/src/views/WorkerManager/WMEditWorkerPool/index.jsx

```diff
@@ -0,0 +1,235 @@
+import { hot } from 'react-hot-loader';
+import React, { Component } from 'react';
+import { bool } from 'prop-types';
+import TextField from '@material-ui/core/TextField';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import MenuItem from '@material-ui/core/MenuItem';
+import CodeEditor from '@mozilla-frontend-infra/components/CodeEditor';
+import ListItem from '@material-ui/core/ListItem';
+import Switch from '@material-ui/core/Switch';
+import { withStyles } from '@material-ui/core';
+import ContentSaveIcon from 'mdi-react/ContentSaveIcon';
+import List from '../../Documentation/components/List';
+import isWorkerTypeNameValid from '../../../utils/isWorkerTypeNameValid';
+import Button from '../../../components/Button';
+import Dashboard from '../../../components/Dashboard';
+
+const gcpConfig = {
+  minCapacity: 0,
+  maxCapacity: 0,
+  capacityPerInstance: 1,
+  machineType: 'n1-highcpu-8',
+  regions: ['us-west2'],
+  userData: {},
+  scheduling: {},
+  networkInterfaces: [{}],
+  disks: [{}],
+};
+const providers = {
+  GCP: 'google',
+};
+
+@hot(module)
+@withStyles(theme => ({
+  successIcon: {
+    ...theme.mixins.successIcon,
+  },
+  createIconSpan: {
+    ...theme.mixins.fab,
+    ...theme.mixins.actionButton,
+  },
+  dropdown: {
+    minWidth: 200,
+  },
+  list: {
+    paddingLeft: 0,
+    paddingRight: 0,
+  },
+  middleList: {
+    paddingTop: theme.spacing.unit * 7,
+    paddingBottom: theme.spacing.unit * 9,
```

Comment:
That's probably a bit too much padding. I think equal padding works but I'll let you decide on that.

Reply:
To my eye, when it's equal, the lower one seems smaller (it's actually a thing about human vision - if you noticed the shape of capital B in many fonts, for example; the upper part is often smaller to make the letter look balanced) - I planned to commit 70 and 90 pixels yesterday, but for some reason the previous iteration ended up in the commit